statemanager/statemanager.go

```diff
@@ -92,3 +93,21 @@ func (s *stateManager) SetStorage(address crypto.Address, key, value binary.Word
 
 	return err
 }
+
+// Bump the sequence number of an account
+func IncSequence(state StateManager, address crypto.Address) error {
```

Comment:
should this be a method of the interface?

Reply:
I tend to like to keep interfaces minimal, and add helpers that compose over the interfaces as separate static functions, that way implementers of `StateManager` get static helper methods like this for free and don't have to reimplement. There may be some context I am missing that makes it better to have this on the interface (or even as a private method, or inline code) though.

Reply:
I initially thought it would make sense as part of the interface too. But I like the simplicity of the state interface doing just the basic operations and it not really needing to know what an account looks like or not.